### ISSUE TITLE
docs.rs build is failling because of stdlib masl compilation

### ISSUE THREAD

**Issue by hackaugusto** [0xPolygonMiden/miden-vm#1208]
These are the relevant log lines from the `docs.rs` build:

```
[INFO] [stderr] error: failed to run custom build command for `miden-stdlib v0.6.0`
[INFO] [stderr] 
[INFO] [stderr] Caused by:
[INFO] [stderr]   process didn't exit successfully: `/opt/rustwide/target/debug/build/miden-stdlib-eca886afdf6dc443/build-script-build` (exit status: 101)
[INFO] [stderr]   --- stdout
[INFO] [stderr]   cargo:rerun-if-changed=asm
[INFO] [stderr]   cargo:rerun-if-changed=../assembly/src
[INFO] [stderr] 
[INFO] [stderr]   --- stderr
[INFO] [stderr]   thread 'main' panicked at /opt/rustwide/cargo-home/registry/src/index.crates.io-6f17d22bba15001f/miden-stdlib-0.6.0/build.rs:62:36:
[INFO] [stderr]   called `Result::unwrap()` on an `Err` value: Os { code: 30, kind: ReadOnlyFilesystem, message: "Read-only file system" }
[INFO] [stderr]   note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
[INFO] running `Command { std: "docker" "inspect" "747083e6431f405446f9cd3cac138e0777497f1d13cc009914f725d839374e2e", kill_on_drop: false }`
[INFO] running `Command { std: "docker" "rm" "-f" "747083e6431f405446f9cd3cac138e0777497f1d13cc009914f725d839374e2e", kill_on_drop: false }`
[INFO] [stdout] 747083e6431f405446f9cd3cac138e0777497f1d13cc009914f725d839374e2e
```

Currently failing at this line:

https://github.com/0xPolygonMiden/miden-vm/blob/c097af7ca53da2026cec10d9245a006b42186643/stdlib/build.rs#L62

**Comment by hackaugusto** [comment]
Maybe instead of deleting the folder, deleting only the folder's content would work?

**Comment by bobbinth** [comment]
Do you mean running `cargo doc` fails? I just tried this on the `next` branch and seems to work (gives a bunch of warnings though).

**Comment by mFragaBA** [comment]
And are MASM docs suposed to get built in the `docs.rs` site?

**Comment by bobbinth** [comment]
No - we don't need MASM docs built in `docs.rs` site (they are mostly for Github).

**Comment by bobbinth** [comment]
Closed by #1341.
